src/runtime/smart-button-api.js

```diff
@@ -80,6 +81,7 @@ export class SmartSubscriptionButtonApi {
       'publicationId': this.deps_.pageConfig().getPublicationId(),
       'theme': this.options_ && this.options_.theme || 'light',
       'lang': this.options_ && this.options_.lang || 'en',
+      'analyticsRequest': analyticsRequest,
```

Comment:
I'd send analyticsContext here.

this.deps_.analytics().getAnalyticsContext().then(context => {
    // use context
}

And this way the iframe has context. When the iframe sends the ready() signal is when it has rendered the smart box. The event should be generated from boq-web iframe.

Reply:
Done.